src/components/OngoingIncidents/OngoingIncidents.css

```diff
@@ -35,4 +37,5 @@ p {
 
 a {
   text-decoration: none;
+  color: cornflowerblue;
 }
```

Comment:
This is a simple fix to remove that old-school html styling from all our nav links. We can easily change to another color/font for the final project by editing this object. Seems to affect all <a> tags though?